src/test/java/teammates/storage/api/FeedbackResponsesDbTest.java

```diff
@@ -335,7 +375,7 @@ public void testGetFeedbackResponses() {
         ______TS("non-existent response");
 
         assertNull(frDb.getFeedbackResponse(expected.getFeedbackQuestionId(), "student1InCourse1@gmail.tmt",
-                                            "student3InCourse1@gmail.tmt"));
+                "student3InCourse1@gmail.tmt"));
```

Comment:
Any reason for this too?

Reply:
This is me over-linting, I reverted this change.